src/Groups/GAPGroups.jl

```diff
@@ -842,7 +1053,7 @@ end
 """
     relators(G::FPGroup)
 
-Return a list of relators for the finitely presented group, i.e. elements `[x_1, ... , x_n]` in `F` = `free_group(ngens(G))` such that `G = F/[x_1,...,x_n]`.
+Return an array of relators for the finitely presented group, i.e. elements `[x_1, ... , x_n]` in `F` = `free_group(ngens(G))` such that `G = F/[x_1,...,x_n]`.
```

Comment:
Wrap long line?

Also, we have just switch most uses of Array to the type aliases Vector and Matrix; so perhaps we should say "a vector" here? Also in a few other places where we mention arrays?

Reply:
Yes.
(My idea was to replace the GAP style "list" by Julia's "array"; but of course "vector" is more specific.)